package.json

```diff
@@ -49,5 +51,30 @@
     "undef": true,
     "white": true,
     "mocha": true
-  }
+  },
+  "gitHead": "88a104c7e83bd9019466273c59d794c7c6ea122f",
+  "bugs": {
+    "url": "https://github.com/achingbrain/mongoose-crate-s3/issues"
+  },
+  "_id": "mongoose-crate-s3@1.1.1",
+  "_shasum": "79039939b81c7453e8d9f7a4b8c098c7bcc5fc71",
+  "_from": "mongoose-crate-s3@~1.1.1",
+  "_npmVersion": "2.8.3",
+  "_nodeVersion": "1.8.1",
+  "_npmUser": {
+    "name": "achingbrain",
+    "email": "alex@achingbrain.net"
+  },
+  "maintainers": [
+    {
+      "name": "achingbrain",
+      "email": "alex@achingbrain.net"
+    }
+  ],
+  "dist": {
+    "shasum": "79039939b81c7453e8d9f7a4b8c098c7bcc5fc71",
+    "tarball": "http://registry.npmjs.org/mongoose-crate-s3/-/mongoose-crate-s3-1.1.1.tgz"
+  },
+  "directories": {},
+  "_resolved": "https://registry.npmjs.org/mongoose-crate-s3/-/mongoose-crate-s3-1.1.1.tgz"
```

Comment:
There's a lot of noise here - looks like you've taken the generated package.json file from an npm download.